vm/interpreter.cpp

```diff
@@ -1032,6 +1033,20 @@ Interpreter::visitSTRADJUST_PRI()
   return true;
 }
 
+bool
+Interpreter::visitBREAK()
+{
+  if (!env_->get()->consoledebugger()->IsEnabled())
+    return true;
+  
+  int err = InvokeDebugger(rt_->GetBaseContext());
+  if (err != SP_ERROR_NONE) {
+    cx_->ReportErrorNumber(err);
```

Comment:
In the new error handling model, it's better for the debugger callback to actually report the error on its own, and for the interpreter to check Environment::hasPendingException(). See invokeNative() for an example.